cves/CVE-2009-3548.yml

```diff
@@ -74,8 +81,8 @@ incomplete_fix_instructions: |
   Please list any fix commits for this vulnerability that had to be corrected
   at a later date.
 incomplete_fixes:
-  - commit:
-    note:
+  - commit: 
+    note: 
   - commit:
     note:
 
```

Comment:
I kinda like this vulnerability. I'll give it 2 upvotes

Reply:
Thank you!